test/rekt/features/kafkasource/readiness.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kafkasource
+
+import (
+	"knative.dev/eventing/test/rekt/resources/svc"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/reconciler-test/pkg/feature"
+
+	"knative.dev/eventing-kafka/test/rekt/resources/kafkasource"
+)
+
+// KafkaSourceGoesReady returns a feature testing if a KafkaSourceGoesReady becomes ready.
+func KafkaSourceGoesReady(name string, cfg ...kafkasource.CfgFn) *feature.Feature {
```

Comment:
Is this function used anywhere? I don't see it in this PR.

Reply:
you're right, it's not used yet. It should probably be added to the smoke tests.

Reply:
Would you like to add this later or in this PR? Just to make it clear and be able to move ahead.

Reply:
I can add this later.